docs/source/parquet.mdx

```diff
@@ -163,4 +163,58 @@ Big datasets are partitioned into Parquet files (shards) of about 500MB each. Th
  "failed": []}
 ```
 
-To read and query the Parquet files, take a look at the [Query datasets from Datasets Server](parquet_process) guide.
\ No newline at end of file
+To read and query the Parquet files, take a look at the [Query datasets from Datasets Server](parquet_process) guide.
+
+## Using the Hugging Face Hub API
+
+For convenience, you can directly use the Hugging Face Hub `/api/parquet` endpoint which returns the list of Parquet URLs.
+
+It takes the dataset name, the configuration name and the split as parameters:
+
+<inferencesnippet>
+<python>
+```python
+import requests
+headers = {"Authorization": f"Bearer {API_TOKEN}"}
+API_URL = "https://huggingface.co/api/datasets/duorc/parquet/ParaphraseRC/train"
+def query():
+    response = requests.get(API_URL, headers=headers)
+    return response.json()
+urls = query()
+```
+</python>
+<js>
+```js
+import fetch from "node-fetch";
+async function query(data) {
+    const response = await fetch(
+        "https://huggingface.co/api/datasets/duorc/parquet/ParaphraseRC/train",
+        {
+            headers: { Authorization: `Bearer ${API_TOKEN}` },
+            method: "GET"
+        }
+    );
+    const urls = await response.json();
+    return urls;
+}
+query().then((response) => {
+    console.log(JSON.stringify(response));
+});
+```
+</js>
+<curl>
+```curl
+curl https://huggingface.co/api/datasets/duorc/parquet/ParaphraseRC/train \
+        -X GET \
+        -H "Authorization: Bearer ${API_TOKEN}"
+```
+</curl>
+</inferencesnippet>
+
+The endpoint response is a JSON containing a list of the dataset's files URLs in the Parquet format. For example, the [`duorc`](https://huggingface.co/datasets/duorc) dataset has one Parquet file for the train split of  "ParaphraseRC" configuration (see the [List splits and configurations](./splits) guide for more details about splits and configurations). 
+
+```json
+["https://huggingface.co/datasets/duorc/resolve/refs%2Fconvert%2Fparquet/ParaphraseRC/duorc-train.parquet"]
```

Comment:
"Funny" that it currently shows a list of three splits. Thanks for reporting this bug on moonlanding, I'll be fixing it. Maybe hold this PR meanwhile.